equinix_metal.tf

```diff
@@ -21,44 +21,44 @@ data "template_file" "user_data" {
   }
 }
 
-resource "metal_project" "new_project" {
-  count           = var.metal_create_project ? 1 : 0
+resource "equinix_project" "new_project" {
+  count           = var.equinix_create_project ? 1 : 0
 
-  name            = var.metal_project_name
-  organization_id = var.metal_organization_id
+  name            = var.equinix_project_name
+  organization_id = var.equinix_organization_id
 }
 
-resource "metal_device" "control_plane" {
+resource "equinix_device" "control_plane" {
```

Comment:
This should be [`equinix_metal_device`](https://registry.terraform.io/providers/equinix/equinix/latest/docs/resources/equinix_metal_device)